iOS/Projects/Features/Record/Sources/Data/DTO/RecordResponseDTO.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  RecordResponseDTO.swift
+//  RecordFeature
+//
+//  Created by 안종표 on 11/30/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - RecordResponseDTO
+
+struct RecordResponseDTO {
+  let workoutTime: Int?
+  let distance: Int?
+  let calorie: Int?
+  let avgHeartRate: Int?
+  let minHeartRate: Int?
+  let maxHeartRate: Int?
+  let createdAt: String?
+  let workout: String?
+}
+
+// MARK: Codable
+
+extension RecordResponseDTO: Codable {}
+
+extension Record {
+  init?(dto: RecordResponseDTO) {
+    guard let workout = dto.workout,
+          let distance = dto.distance,
+          let createdAt = dto.createdAt,
+          let workoutTime = dto.workoutTime,
+          let (startTime, endTime) = Record.timeToTime(createdAt: createdAt, workoutTime: workoutTime)
+    else {
+      return nil
+    }
+    mode = workout
+    self.distance = distance
+    self.startTime = startTime
+    self.endTime = endTime
+  }
+
+  private static func timeToTime(createdAt: String, workoutTime: Int) -> (startTime: String, endTime: String)? {
+    let startTime = createdAt.components(separatedBy: " ")[1]
```

Comment:
p3: 이미 `.whitespaces`가 빌트인으로 존재합니다 :)
```suggestion
    let startTime = createdAt.components(separatedBy: .whitespaces)[1]
```